teacher.rb

```diff
@@ -0,0 +1,15 @@
+# teacher.rb
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(id, specialization:, age: 18, name: 'Unknown', parent_permission: true)
+    super(id, age: age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] The parameter `age` should not be an optional parameter, it should be required.

- [ ] **[OPTIONAL]** If you took the suggestion in the previous comment then you won't need to pass in IDs and track the previous IDs.

  ```
  def initialize(specialization:, age:, name: 'Unknown', parent_permission: true)
    super(age: age, name: name, parent_permission: parent_permission)
  ```